wp-content/plugins/tapestry/templates/single-tapestry.php

```diff
@@ -34,7 +34,6 @@
 			<script src="<?php echo plugin_dir_url(__FILE__) ?>tapestry-d3/libs/h5p-resizer.min.js" charset="UTF-8"></script>
 
             <script>
-				var jsonUrl = "<?php echo plugin_dir_url(__FILE__) ?>tapestry-d3/tapestry.json";
```

Comment:
Please pass the URL into the JS the same way so that we don't have to hardcode it in. Something like 
`var apiUrl = "<?php get_rest_url(null, 'wp-json/tapestry-tool/v1'); ?>";` should work

Reply:
Good idea, using the get_rest_url function now.